documentation/source/installation.rst

```diff
@@ -0,0 +1,116 @@
+Installation
+============
+
+Configuring SSH keys
```

Comment:
As we talked about yesterday. I agree that the ssh stuff doesnt need to live here but maybe we should link to it if it is put somewhere else because I can see questions coming up about it